gnome-initial-setup/pages/summary/gis-summary-page.ui

```diff
@@ -19,12 +19,81 @@
             <property name="valign">center</property>
             <property name="orientation">vertical</property>
             <property name="spacing">12</property>
-            <child type="center">
-              <object class="GtkImage" id="image">
+            <child>
+              <object class="GtkBox" id="summary_contents_box">
                 <property name="visible">True</property>
-                <property name="resource">/org/gnome/initial-setup/ready-to-go.svg</property>
-                <property name="width_request">96</property>
-                <property name="height_request">96</property>
+                <property name="halign">center</property>
+                <property name="valign">center</property>
+                <property name="orientation">vertical</property>
+                <property name="spacing">12</property>
+                <child type="center">
+                  <object class="GtkImage" id="image">
+                    <property name="visible">True</property>
+                    <property name="resource">/org/gnome/initial-setup/ready-to-go.svg</property>
+                    <property name="width_request">96</property>
```

Comment:
The ada.png asset is 350x350 – is the size it comes out as correct?

Reply:
Yeah. It takes the images' size, so it looks fine.